designs/administrator/olm/data-list-operator-hub/index.md

```diff
@@ -0,0 +1,24 @@
+---
+parent: Administrator
+version: 4.6
+---
+
+# List view on OperatorHub
+
+In this story the list view demonstrates the Operator tiles in a new light by allowing the user to view the Operators in a list view. The list view shows more operators in a list view in order to expose more information to the user in a readable way.
+
+When the user selects the list view the PatternFly data-list component powers more compact styling so more information can be viewed at once.
+
+![Data list screen one](img/data-list-screen-operatorhub.jpg)
+
+## See more badges
+
+The user clicks into the 3+ badge to see more badges when there are conditional badges selected. Whatever number of conditional filters are selected should be represented in the "more" badge shown on this screen as an example. This badge will not appear if 2 badges or less appear on an Operator.
+
+![Click to see more](img/see-more.jpg)
```

Comment:
Yeah, let me play around with that, I like that idea!

Reply:
@dmesser I looked into Liz's recommendation to change the badge position and this this will require some custom CSS that we will apply on the PatternFly component to change the arrow position, which would ultimately look like this: 

<img width="454" alt="Screen Shot 2020-06-15 at 10 35 48 AM" src="https://user-images.githubusercontent.com/40275806/84678044-8a642800-aefd-11ea-86f8-9feb285b8c33.png">

Do you have any concerns about applying custom CSS to the PatternFly component to change the arrow position or is this feasible for us to implement today?

Reply:
@rachael-phillips The arrow should actually be ok. The popover for the yaml shortcuts will shift based on location on the page. Here is a screenshot: 
![Screen Shot 2020-06-15 at 12 15 22 PM](https://user-images.githubusercontent.com/35978579/84681497-30b22c80-af02-11ea-951c-031c4f81b3a5.png)

Reply:
That's awesome! Thank you! :D